### ISSUE TITLE
Host name did not verify for redmine behind https 

### ISSUE THREAD

**Issue by pokebadgerswithspoon** [redminenb/redminenb#34]
If redmine installation is behind https then it can not be added even though ssl certificate is valid.
![redminenb](https://cloud.githubusercontent.com/assets/6481705/7414187/a315688c-ef59-11e4-9143-7bea210e3eb4.PNG)

**Comment by pokebadgerswithspoon** [comment]
Is there reason for `RedmineManagerFactoryHelper` to setup a custom hostnameVerifier?
If lines 
```java
    socketFactory.setHostnameVerifier(new X509HostnameVerifier() {
        ....
        });
```
removed then the problem is gone

**Comment by matthiasblaesing** [comment]
To answer the last question first:

The hostname verifier is installed to support the usage of "SSL Certificate Exception" module, which then allows usage of SSL certificates, that are self-signed or created for different hostnames. In this case this blows, because the "defaultHostnameVerifier" in java just rejects hostnames.

So I'll look into a solution, but I'm not sure yet - looks like I need to reimplement the whole hostname check algorithm *arg*.

**Comment by matthiasblaesing** [comment]
I would appreciate it, if you could give this pre-release version a spin:

https://github.com/redminenb/redminenb/wiki/redminenb-1.8.1-SNAPSHOT.nbm

It contains a fixed hostname verifier.

**Comment by matthiasblaesing** [comment]
Will do release now - thanks for reporting, testing would have been nice.

**Comment by pokebadgerswithspoon** [comment]
Thanks for the update.
Just got SNAPSHOT installed. Works as expected:
* accepts certificate of  https://google.com
* accepts home made certificate when the certificate is accepted by OS
* self signed certificate not tested

**Comment by matthiasblaesing** [comment]
Thank you for checking! The fix is part of the 1.9 release.
